packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts

```diff
@@ -33,6 +33,16 @@ async function _getClientVersion(config: EngineConfig) {
     const [version] = engineVersion.split('-') ?? []
     const [major, minor, patch] = version.split('.')
 
+    // If the engines version is 4.17
+    // This is for testing Prisma 5 dev versions in ecosystem-tests against a real Data Proxy
+    // We need to test with a Data Proxy engine version deployed
+    // which includes an engine with `jsonProtocol` GA
+    // because `jsonProtocol` is GA in the client
+    // but not in the Data Proxy version that we fallback to by default, here `4.16.2`
```

Comment:
```suggestion
    // If the engines version is 4.17, this is for testing Prisma 5 dev versions in
    // ecosystem-tests against a real Data Proxy. We need to test with a Data Proxy
    // engine version deployed which includes an engine with `jsonProtocol` GA
    // because `jsonProtocol` is GA in the client but not in the Data Proxy version
    // that we fallback to by default, here `4.16.2`
```

(stylistic change)